src/test/java/com/segment/analytics/android/integration/firebase/FirebaseTest.java

```diff
@@ -1,14 +1,18 @@
 package com.segment.analytics.android.integration.firebase;
 
 import android.content.Context;
+import android.content.pm.ActivityInfo;
```

Comment:
Also not sure what these two new packages are - looks as if they're not used anywhere.

Reply:
@brennan yeah it looks like it is part of the `android.content` package import. When I delete them the tests won't compile :/ 
<img width="346" alt="Screen Shot 2020-02-20 at 2 30 59 PM" src="https://user-images.githubusercontent.com/31782219/74986052-ac391c80-53ed-11ea-933b-776ad3a98eef.png">